### PR TITLE
Normalize version strings in changelog entries

### PR DESCRIPTION
**Pull request by jimwins**
Now they're always `N.X.Y` for a PHP version, or `PECL extension N.X.Y` for PECL extensions.

### CODE REVIEW

File: reference/memcache/memcache/delete.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   <methodsynopsis>
    <type>bool</type><methodname>Memcache::delete</methodname>
    <methodparam><type>string</type><parameter>key</parameter></methodparam>
-   <methodparam choice="opt"><type>int</type><parameter>timeout</parameter><initializer>0</initializer></methodparam>
+   <methodparam choice="opt"><type>int</type><parameter>exptime</parameter><initializer>0</initializer></methodparam>
   </methodsynopsis>
```
Comment (Girgias) [inline reference/memcache/memcache/delete.xml:15]: This doesn't seem related to the normalization, split into a new PR?

File: reference/mongodb/bson.xml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <!-- $Revision$ -->
 
  <book xml:id="book.bson" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <?phpdoc extension-membership="pecl" ?>
```
Comment (Girgias) [inline reference/mongodb/bson.xml:5]: Please merge the commit that does this already on its own

Comment (jimwins) [reply reference/mongodb/bson.xml:5]: Done.

File: reference/memcache/memcache/delete.xml (hunk 1 above)
Comment (jimwins) [reply reference/memcache/memcache/delete.xml:15]: Split out into #3687.

**Comment by jimwins** [comment]
Okay, PR is just a clean set of changes to versions now. [the change to extension-membership for the MongoDB books](https://github.com/php/doc-en/pull/3672/commits/a51ebd074abdc1cd18ef9da1628d6d5049616562) was already cherry-picked and pushed into master.

File: reference/intl/dateformatter/set-calendar.xml [hunk 3]
```diff
@@ -87,7 +87,7 @@
      </thead>
      <tbody>
       <row>
-       <entry>5.5.0/PECL 3.0.0</entry>
+       <entry>5.5.0</entry>
        <entry>
```
Comment (Girgias) [inline reference/intl/dateformatter/set-calendar.xml:91]: Either remove the whole changelog as we don't care about PHP 5 anymore or only keep the PECL 3.0.0 as you've kept it for the previous file.

**Review by Girgias — CHANGES_REQUESTED** [review]

**Review by Girgias — APPROVED** [review]
Thank you!

**Pull request merged**